### ISSUE TITLE
Markdown output contains HTML comments

### ISSUE THREAD

**Issue by sigtm** [HcySunYang/vuese#92]
Maybe I'm missing something, but when using `genType: 'markdown'` it outputs `.md` files with comments like these:

`<!-- @vuese:Card:props:start -->`

Isn't this non-standard for markdown? We use `markdown-it` to render the content, which simply renders it as plain text wrapped in a `<p>...</p>`.

Apologies if I'm just missing something. I'm testing Vuese for the first time, and it's looking good so hoping to be able to use it to document our component library.

**Comment by HcySunYang** [comment]
Yes, this is a comment generated by vuese to mark a specific block so that vuese can insert the content into the right place.

**Comment by terazus** [comment]
Hi guys,
These comments break my readthedoc build.
Any idea on how to get them removed automatically?
Thx

edit: actually, adding a new line after each comment, readthedoc can read the markdown.
So:

``` 
<!-- @vuese:Record:methods:start -->

|Method|Description|Parameters|
|---|---|---|
|getTitle|Method to build and return the page title to be included as a metadata|-|
````
is valid but not
```
<!-- @vuese:Record:methods:start -->
|Method|Description|Parameters|
|---|---|---|
|getTitle|Method to build and return the page title to be included as a metadata|-|
```

edit 2:
Im actually starting to think this is a problem with sphinx and not vuese !

**Comment by adjenks** [comment]
This is the one thing I really want fixed. My markdown renderer has HTML disabled for safety reasons, so the comments are displayed and it looks ugly.

**Comment by michelegirini** [comment]
hi all, opened pull request https://github.com/vuese/vuese/pull/218 that add a \n after the html comment, this way markdown could be easily interpreted (even on trello, with the right browser plugin)

**Comment by IWANABETHATGUY** [comment]
Closed due to #218, feel free to reopen it.
